examples/deployments/uc_functions/run.py

```diff
@@ -0,0 +1,141 @@
+"""
+# Set environment variables
+export DATABRICKS_HOST="..."
+export DATABRICKS_TOKEN="..."
+export DATABRICKS_WAREHOUSE_ID="..."
+export OPENAI_API_KEY="..."
+export MLFLOW_ENABLE_UC_FUNCTIONS=true
+
+# Run server
+mlflow deployments start-server \
+    --config-path examples/deployments/deployments_server/openai/config.yaml --port 7000
+
+# Run client
+python examples/deployments/uc_functions/run.py
+"""
+
+import openai
+
+client = openai.OpenAI(base_url="http://localhost:7000/v1")
+
+req = {
+    "messages": [
+        {
+            "role": "user",
+            "content": (
+                "I opened a bank account on 2024/01/05 and I have to keep my deposit there "
+                "for 90 days to qualify a bonus. When can I withdraw the money?"
+            ),
+        }
+    ],
+}
+
+# Without tools
+print("--- Without tools ---")
+resp = client.chat.completions.create(
+    model="chat",
+    **req,
+)
+print(resp.choices[0].message.content)
+
+# With tools
+print("--- With tools ---")
+resp = client.chat.completions.create(
+    model="chat",
+    **req,
+    tools=[
+        {
+            "type": "uc_function",  # type: ignore
+            "uc_function": {
+                "name": "ml.haru.add",
```

Comment:
In the README, we need to demonstrate to users how they can create their own UC functions for tool calling. We shouldn't hardcode a specific function here.

Reply:
agreed, will add a function in README for demonstration